documentation/examples/prometheus.yml

```diff
@@ -27,3 +27,6 @@ scrape_configs:
 
     static_configs:
       - targets: ["localhost:9090"]
+        # The label name is added as a label `label_name=<label_name>` to any timeseries scraped from this config.
```

Comment:
nit: the bit after `=` should say value not name.

Reply:
Thanks for your suggestion! So my intention was to add a label as a parameter for an exporter so that it can be used for filtering both in prometheus and grafana dashboards.